js-basics/js-variables-and-numbers_simple-calculator/js/index.js

```diff
@@ -59,26 +71,34 @@ const multiplyByTwoButton = document.querySelector(
 );
 const divideByTwoButton = document.querySelector('[data-js="divide-by-two"]');
 
+let newOperand1 = operand1;
+
 increaseByOneButton.addEventListener("click", () => {
   // 7. Increase the value of "operand1" by 1. Log the new value to the console.
+  console.log(++newOperand1);
```

Comment:
If you directly increase the operand variables then they are affected by other functions. You can declare a variable and assign the operand to it, this way it won't be affected outside of the function